lib/cog_api/http/base.ex

```diff
@@ -115,7 +114,7 @@ defmodule CogApi.HTTP.Base do
   end
 
   defp make_url(%Endpoint{proto: proto, host: host, port: port,
-                            version: version}, route, params \\ %{}) do
+                            version: version}, route, params_or_id \\ %{}) do
```

Comment:
(And we won't need the additional conditional (pattern matching) in `format_params`_